src/big/big_float.cr

```diff
@@ -8,10 +8,23 @@ struct BigFloat < Float
   include Comparable(BigFloat)
   include Comparable(Float)
 
+  # Creates a `BigFloat` with the value zero.
+  #
+  # ```
+  # require "big"
+  # BigFloat.new.inspect # => 0_big_f
+  # ```
   def initialize
     LibGMP.mpf_init(out @mpf)
   end
 
+  # Creates a `BigFloat` with the value denoted by *str*.
+  #
+  # Raises `ArgumentError` if the string doesn't denote a valid float.
+  #
+  # ```
+  # BigFloat.new("-0.123456789101101987654321") # => -0.123456789101101987654321
```

Comment:
`_big_f` suffix?

Reply:
nope, it's added only in `#inspect`.

Reply:
Yeah, that's what the expression value should show, see the example of the arg-less constructor above.

Reply:
@straight-shoota Don't you like `BigFloat.new.inspect # => 0_big_f` ?

Reply:
I'm not sure if I like it in general, because it looks like a literal but there is none.
However, if you use that format for the value of the arg-less constructor example, this one should be equal.

Reply:
Another reason for removing suffixes... there are non for big numbers. Unless we inspect them to ```BigInt.new("#{string}")``` but that's unreadable and ugly.

Reply:
Good points - fixed!